app/src/main/java/turtler/voyageur/activities/LoginActivity.java

```diff
@@ -74,14 +74,20 @@ public void onCompleted(GraphResponse response) {
                             JSONObject data = picture.getJSONObject("data");
                             pictureUrl = data.getString("url");
                             currentUser.setEmail(email);
-                            currentUser.setUsername(name);
```

Comment:
@caro5 wait which bug? the reason this line was causing an error was because when you use `setUsername`, Parse expects it to be unique. so since we were using it for name, and both my accounts had the same name, it didn't allow us to save the names of the fields properly. this wouldn't happen for `pictureUrl` because Parse doesn't expect it to be unique.